src/gpuhunt/_internal/constraints.py

```diff
@@ -2,6 +2,7 @@
 
 from gpuhunt._internal.models import (
     AcceleratorVendor,
+    AmdArchitecture,
```

Comment:
Please rename it to `AMDArchitecture` for consistent naming (as we already have ALLCAPS `AMDGPUInfo`).